wolfhsm/wh_client.h

```diff
@@ -2493,4 +2493,102 @@ int wh_Client_CertVerifyAcertDma(whClientContext* c, const void* cert,
 
 #endif /* WOLFHSM_CFG_DMA */
 
+/*
+ * @brief Client-side keyId manipulation API
+ *
+ * This section defines the client-facing API for working with key identifiers.
+ * Clients use simple numeric IDs (0-255) with optional flags to indicate
+ * global or wrapped keys. The server translates these to full internal
+ * representations with TYPE/USER/ID fields.
+ *
+ * Client keyId usage:
+ * - Regular keys: Simple numeric ID (e.g., 5)
+ * - Global keys: ID with WH_CLIENT_KEYID_GLOBAL_FLAG set
+ * - Wrapped keys: ID with WH_CLIENT_KEYID_WRAPPED_FLAG set
+ * - Wrapped metadata: Must use full WH_MAKE_KEYID() construction including type
+ *    and metadata when populating the ID field in metadata to be wrapped
+ */
+
+/* Client-facing key flags (temporary, stripped by server during translation) */
+
+/* Bit 8: Client-to-server signal for global key (shared across all clients) */
+#define WH_CLIENT_KEYID_GLOBAL_FLAG ((whKeyId)0x0100)
+
+/* Bit 9: Client-to-server signal for wrapped key */
+#define WH_CLIENT_KEYID_WRAPPED_FLAG ((whKeyId)0x0200)
+
+/* Combined mask of all client-facing flags */
+#define WH_CLIENT_KEYID_FLAGS_MASK \
+    (WH_CLIENT_KEYID_GLOBAL_FLAG | WH_CLIENT_KEYID_WRAPPED_FLAG)
+
```

Comment:
```suggestion

#define WH_CLIENT_KEYID_ID_MASK ((whKeyId)0x00FF)
```
Ensure client ID's are only the bottom bits